src/makeRobotAccountant.js

```diff
@@ -6,7 +6,29 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+  let isOdd = true;
+
+  return function(a) {
+    if (++counter <= 3) {
+      return function(b) {
+        return a + b;
+      };
+    } else {
+      isOdd = !isOdd;
+      counter++;
+
+      if (isOdd) {
```

Comment:
at this part you are setting isOdd to opposite value => false, this if will never work